examples/data-objects/diceroller/src/main.tsx

```diff
@@ -37,7 +34,7 @@ export interface IDiceRollerViewProps {
     model: IDiceRoller;
 }
 
-export const DiceRollerView: React.FC<IDiceRollerViewProps> = (props: IDiceRollerViewProps) => {
+export const DiceRollerView = (props: IDiceRollerViewProps) => {
     const [diceValue, setDiceValue] = React.useState(props.model.value);
```

Comment:
examples/apps/view-framework-sampler/src/views had trouble here, but others were fine. Did the problem spread?
That is the appropriate typing; so we don't want to remove it unless there is something funny.

Reply:
The only check that I did here was that I was able to build the project. Is that what you mean?

Reply:
Sort of. I mean it is unexpected that you need to make any changes. The original file is correct.
Did you make this change because there was an error here or did you make it because that is the change made elsewhere. We should only change the places that need the change.

Reply:
One thing that might be causing random failures is `lerna bootstrap` with the `--hoist` flag. We found through hard experience that when hoisting, the lockfile is _not always_ obeyed. there is more in AB#1047. From that issue:

> You would expect lerna to obey --ci under the hood, and run `npm ci` on our pipelines, but that does not seem to be the case. Instead, it runs `npm install`.

So if there has been a release of, say, @types/react that matches our range in package.json, it might be getting picked up regardless of what the lockfile says.

Reply:
I looked through the packages that were resolved for the issue (not here but in examples/apps/view-framework-sampler/src/views) and couldn't find problem.
Will be nice to move on to pnpm.
On the side while working on the other way to move forward I identified a bunch of the missing dependencies. After this goes in I might apply and see if things clear up with the proper dependencies in place.

Reply:
I did this change following your suggestion [https://teams.microsoft.com/l/message/19:ac1db74d64f74d078d79d06af06114ef@thread.v2/1695662707288?context=%7B%22contextType%22%3A%22chat%22%7D](here)

Reply:
Yes. That suggestion was for exactly view-framework-sampler package where there was a problem. We didn't see it elsewhere; so that kludge shouldn't be applied elsewhere.
Tyler commented that things were building for him with a number of files reverted. Perhaps the root change lead to node_modules weirdness as he was able able to revert the view-framework-sampler code change. See that comment - wants to know if CI passes with those back to original.